docs/_documentations/mdt-vsc-usingadifferenttemplate.md

```diff
@@ -1,24 +1,19 @@
 ---
 layout: docs
-title: Using a template repository of your own
-description: Using a template repository of your own
-keywords: getting started, setting up, projects, update, help, Theia, test, edit, Theia editor, using own IDE, empty page, refresh, credentials, default editor, Node.js profiling support, code highlighting, JavaScript file, template repository
+title: Configuring template sources
+description: Configuring template sources
+keywords: getting started, setting up, projects, update, help, Theia, test, edit, Theia editor, using own IDE, empty page, refresh, credentials, default editor, Node.js profiling support, code highlighting, JavaScript file, template source
 duration: 1 minute
 permalink: mdt-vsc-usingadifferenttemplate
 type: document
 order: 8
 parent: usingcodewind
 ---
 
-# Using a template repository of your own
+# Configuring template sources
 
-When you create a new project, you can choose from the default set of templates available, or you can choose from a template repository of your own. To add a new repository, edit the `.config/repository_list.json` file within your `codewind-workspace`. The file is a json file that contains an array of objects with two fields, the URL of the repository and a description:
+When you create a new project, you can choose from the default set of templates available, or you can choose from a template source of your own. Use Codewind to manage templates. Templates can make application development easier by providing a structure and boilerplate code to help get a new project started.
 
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Manage Template Sources** to add a source to the table.
+- Enable or disable a source type to view the sources of that type in the wizard.
```

Comment:
What does this line mean?

Reply:
maybe "Enable or disable sources to set if that source's templates should appear in the Create Project wizard"

Reply:
Yes, Guy mentioned a wizard. Maybe something like...
```
- Enable a source to make that source template appear in the Create Project wizard.
- Disable a source to remove that source template from the Create Project wizard.
```

Reply:
yes, although i'm not sure something that simple requires so many words to explain :)